scripts/buildLocales.js

```diff
@@ -0,0 +1,31 @@
+import * as fs from 'fs'
+
+// This script loads the current JSON-based locales files and merges them into
+// a single importable ES module for bundling purposes
+const localesRoot = process.argv[2]
+if (!localesRoot) {
+    console.log('usage: buildLocales <locales/feature dir>')
+}
+
+const locales = {}
+const localesDirs = fs.readdirSync(localesRoot).filter(f => !f.startsWith('.'))
+for (const l of localesDirs) {
```

Comment:
Nit:
```suggestion
for (const locale of localesDirs) {
```

Reply:
I actually had that originally but I've seen some odd typo bugs due to stuff like 'file of files' .  But NBD, change made. :)